pkgs/top-level/all-packages.nix

```
@@ -22488,6 +22488,7 @@ in
   pdfsam-basic = callPackage ../applications/misc/pdfsam-basic { };
 
   mupdf = callPackage ../applications/misc/mupdf { };
+  mupdf-1-17 = callPackage ../applications/misc/mupdf/mupdf-1-17.nix { };
```

generally, pinned versions use an underscore `_`
```suggestion
  mupdf_1_17 = callPackage ../applications/misc/mupdf/1.17.nix { };
```

---

PR updated.